core/vm/intpool.go

```diff
@@ -65,3 +68,40 @@ func (p *intPool) put(is ...*big.Int) {
 		p.pool.push(i)
 	}
 }
+
+const poolDefaultCap = 25
+
+// intPoolPool manages a pool of intPools.
+type intPoolPool struct {
+	pools []*intPool
+
+	// Mutex
+	mutex sync.Mutex
```

Comment:
Nitpick, but throughout the code I usually use `lock` and Felix likes to use `mu`. Either is fine, but lets stick to one of those and not start a third naming for it :)